docs/docs/30-administration/22-backends/40-kubernetes.md

```diff
@@ -243,12 +261,24 @@ Determines if `RWX` should be used for the pipeline volume's [access mode](https
 
 Additional labels to apply to worker pods. Must be a YAML object, e.g. `{"example.com/test-label":"test-value"}`.
 
+### `WOODPECKER_BACKEND_K8S_POD_LABELS_ALLOW_FROM_STEP`
+
+> Default: `false`
+
+Determines if additional Pod labels can be defined from a step's backend options.
```

Comment:
Capitalization is used in Kubernetes docs. Don't know if this is convention, though.

Reply:
Yeah, in my experience, the Kubernetes object types (Pod, Secret, Deployment, etc.) are typically capitalized. I'm happy to go through and capitalize the others or change this reference. Just let me know which way you guys would like to go.

Reply:
Ok, I went through capitalized all of the standard Kubernetes object names I could find. I think this will make it feel closer to the Kubernetes documentation since we're following the same conventions.